datadog/tests/resource_datadog_application_key_test.go

```diff
@@ -53,7 +53,7 @@ func TestAccDatadogApplicationKey_Update(t *testing.T) {
 				Config: testAccCheckDatadogApplicationKeyConfigScopesRequired(applicationKeyName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckDatadogApplicationKeyExists(accProvider, resourceName),
-					resource.TestCheckResourceAttr(resourceName, "scopes", applicationKeyScopes),
+					resource.TestCheckResourceAttr(resourceName, "scopes.#", fmt.Sprint(len(applicationKeyScopes))),
```

Comment:
would love to understand this change so that I can do better with future PRs

Reply:
The TestCheckResourceAttr function only supports a string value to compare to, and the sdk doesn't have great support for list or set equality checking, only checking for specific elements within the list. Our workaround is to just check the length of the list.

Also, I wasn't able to push because the original PR was from a fork of this repository.